src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final double firstLeg;
+    private final double secondLeg;
+    private final Color color;
+
+    public RightTriangle(Color color, double firstLeg, double secondLeg) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. Instead of passing 'color' directly to the constructor, you should use a 'ColorSupplier' to get a random color.